google-account-plugin/src/com/google/gct/idea/AccountPluginInitializationComponent.java

```diff
@@ -38,30 +31,43 @@ public String getComponentName() {
 
   @Override
   public void initComponent() {
-    if (!"AndroidStudio".equals(PlatformUtils.getPlatformPrefix())) {
-      FeedbackUtil.enableGoogleFeedbackErrorReporting(PLUGIN_ID);
+    if (PlatformInfo.SUPPORTED_PLATFORMS.contains(PlatformUtils.getPlatformPrefix())) {
+      enableFeedbackUtil();
+      if (!ApplicationManager.getApplication().isUnitTestMode()) {
+        enableUsageTracking();
+      }
     }
 
-    if (ApplicationManager.getApplication().isUnitTestMode()) {
-      return;
-    }
 
-    // For Google Usage Tracker
-    // Ensure that the notification manager (also an application component) is registered first;
-    // otherwise this component's initComponent() call will fire a notification event bus
-    // to show the opt-in dialog, but the notification component may not yet have been initialized
-    // and is therefore not subscribed and listening.
+  }
+
+  @Override
+  public void disposeComponent() {
+    // no-op
+  }
+
+  @VisibleForTesting
+  void enableFeedbackUtil() {
+    FeedbackUtil.enableGoogleFeedbackErrorReporting(PLUGIN_ID);
```

Comment:
It was kind of evil of me to make this a Util class with a static method to enable feedback.